recipe/build.sh

```diff
@@ -13,6 +13,9 @@ set -x -e
 INCLUDE_PATH="${PREFIX}/include"
 LIBRARY_PATH="${PREFIX}/lib"
 
+# Always build PIC code for easier static linking
+CXXFLAGS="${CXXFLAGS} -O3 -fPIC"
```

Comment:
It shouldn't be needed by `-fPIC`.

That is a good question about `-O3`. I don't know how well that works with Boost. Have seen other libraries that have issues with this level of optimization. We should investigate this further.